src/app/pages/storage/volumes/permissions/containers/permissions-sidebar/permissions-sidebar.component.ts

```diff
@@ -55,6 +55,17 @@ export class PermissionsSidebarComponent implements OnInit, OnChanges {
         this.isLoading = state.isLoading;
         this.acl = state.acl;
         this.stat = state.stat;
+        if (this.acl) {
+          for (const acl of this.acl.acl) {
+            if (acl.tag.toLowerCase().includes('owner') && acl.who == null) {
```

Comment:
It's better to rely on enum values instead of parsing strings. This is fragile in the future.